azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/executingflowpage.vm

```diff
@@ -121,10 +121,19 @@
       <li id="jobslistViewLink"><a href="#jobslist">Job List</a></li>
       <li id="flowLogViewLink"><a href="#log">Flow Log</a></li>
       <li id="statsViewLink"><a href="#stats">Stats</a></li>
-      #if ($executionExternalLinkURL)
-        <li><a id="analyzerButton" href="${executionExternalLinkURL}" class="btn btn-info btn-sm"
-               type="button" target="_blank"
-               title="Analyze job in ${executionExternalLinkLabel}">${executionExternalLinkLabel}</a>
+      #foreach ($analyzer in $externalAnalyzers)
+        #if (!${analyzer.isLinkUrlValid()})
+          #set($title = "Execution is not analyzable in ${analyzer.getLabel()} at the moment. "
+          + "If ${analyzer.getLabel()} is applicable for this execution, you can try again later.")
+          #set($isDisabled = "disabled")
+        #else
+          #set($title = "Analyze execution in ${analyzer.getLabel()}")
+          #set($isDisabled = "")
+        #end
+        #set($id = "analyzerButton${analyzer.getTopic()}")
+        <li><a id="${id}" href="${analyzer.getLinkUrl()}" class="btn btn-info btn-sm btn-external"
+               type="button" ${isDisabled} target="_blank"
+               title="${title}">${analyzer.getLabel()}</a>
         </li>
```

Comment:
To render the button disabled, let's add `disabled` as a css class instead, like this:
<a id="${id}" href="${analyzer.getLinkUrl()}" class="btn btn-info btn-sm btn-external **${isDisabled}**" type="button" target="_blank" title="${title}">${analyzer.getLabel()}</a>

then, for the styles we'll only need:
.btn-external.disabled {
     pointer-events: auto; //I think this should be none instead since there will be no redirection
     cursor: not-allowed;
   }

Reply:
Hi
I tried, if I use this, tooltip won't be showed up on disabled button.

Reply:
You're right, if we set `pointer-events: none;` then the tooltip won't be displayed so let's use your original value: auto. Could you make the change to disable the button using a CSS class?